velox/functions/sparksql/aggregates/tests/CollectSetAggregateTest.cpp

```diff
@@ -260,5 +260,18 @@ TEST_F(CollectSetAggregateTest, rowWithNestedNull) {
       {data}, {}, {"collect_set(c0)"}, {"spark_array_sort(a0)"}, {expected});
 }
 
+TEST_F(CollectSetAggregateTest, nullType) {
+  auto data = makeRowVector({
+    makeNullConstant(TypeKind::UNKNOWN, 3),
+  });
+
+  auto expected = makeRowVector({
+      makeArrayVectorFromJson<int32_t>({"[]"}),
+  });
+
+  testAggregations(
+      {data}, {}, {"collect_set(c0)"}, {}, {expected});
```

Comment:
Would you add some tests for null type being the grouping key?

Reply:
add one more test